### PR TITLE
Starting on the Fix Pack

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/4198

### CODE REVIEW

File: release_notes/fixpack.adoc [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+ 
+. Upgraded the `oc` and `kubectl` CLIs to the latest versions for the Visual Web Terminal.
+. Administrator (`admin`) role access to the pod logs of managed clusters is fixed.
+. The product uninstallation process was improved.
+. Added a label for `Bare metal` to the Cloud field options list, on the _Importing a cluster_ page.
+. The default `Network type` when you create a cluster is updated from OpenShiftSDN to OVNKubernetes.
+. Subscriptions support `kustomization.yaml` files that contains an inline patch where the patch content inside the file is a single string.
+. Improved how cloud providers manage sensitive data. 
+. Removed DNS virtual IP parameter from the create cluster flow. (https://bugzilla.redhat.com/show_bug.cgi?id=1866513[BZ 1862506])
+// ^ check with doc team
```
Comment (swopebe) [inline release_notes/fixpack.adoc]: Referenced on Slack. See previous issue and discussion. 

If BZ is reported and accessed by external user, this makes sense. But if not, it is for internal-use only and not required for the user. We don't want to insert them just to insert them. Last time **all of this was internally reported** so it did not make sense to add internal bugs that user would not need to see. Also, we keep our list pretty basic by design compared to OCP. (By design meaning architects and such wanted a simple list, like the one you created, which is awesome!)

(And I know you already do this, just getting it doc'ed for future @dockerymick :-)! We look at the **GH issues** mostly to eliminate the known issues, and those come from the BZ for the ACM team.)

File: release_notes/fixpack.adoc [hunk 2]
```diff
@@ -1,4 +1,15 @@
 [#fix-pack-updates]
 = Fix pack updates
 
-There is currently no fix pack for this version of the product. Fixes are listed when fix packs are released.
+View a summarized list of {product-title} fix pack 2.0.1 updates.
```
Comment (swopebe) [inline release_notes/fixpack.adoc]: Looks great!!!

File: release_notes/fixpack.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/fixpack.adoc]: @bcpswope Thanks Brandi, I will remove the link to BZ bc i think the link that I provided is internal. I also checked Errata and did not see the Bug yet: https://access.redhat.com/errata/#/?q=&p=1&sort=portal_publication_date%20desc&rows=10&portal_product=Red%20Hat%20Advanced%20Cluster%20Management%20for%20Kubernetes

File: release_notes/fixpack.adoc [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+ 
+. Upgraded the `oc` and `kubectl` CLIs to the latest versions for the Visual Web Terminal.
+. Administrator (`admin`) role access to the pod logs of managed clusters is fixed.
+. The product uninstallation process was improved.
+. Added a label for `Bare metal` to the Cloud field options list, on the _Importing a cluster_ page.
+. The default `Network type` when you create a cluster is updated from OpenShiftSDN to OVNKubernetes.
+. Subscriptions support `kustomization.yaml` files that contains an inline patch where the patch content inside the file is a single string.
+. Improved how cloud providers manage sensitive data. 
+. Removed DNS virtual IP parameter from the create cluster flow.
+. Overview page continues to display cluster details even after managed clusters are detached.
```
Comment (jnpacker) [inline release_notes/fixpack.adoc]: Lets change it to this: Overview page will not become blank when cluster are detached.

File: release_notes/fixpack.adoc [hunk 4]
```diff
@@ -1,4 +1,14 @@
 [#fix-pack-updates]
 = Fix pack updates
 
-There is currently no fix pack for this version of the product. Fixes are listed when fix packs are released.
+View a summarized list of {product-title} fix pack 2.0.1 updates.
+
```
Comment (jnpacker) [inline release_notes/fixpack.adoc:5]: We should add a bullet. Improvements and fixes to the cluster import process

**Pull request merged**